CODEOWNERS

```
@@ -1,6 +1,6 @@
 # Owners of the repository
-* @jingtang10 @yigit @deepankarb @kevinmost @aditya-07 @stevenckngaa
+* @jingtang10 @yigit @deepankarb @kevinmost @aditya-07 @stevenckngaa @ktarasenko
 
 # Owners of SDC library and gallery app
-datacapture/ @jingtang10 @yigit @deepankarb @joiskash @kevinmost @aditya-07 @stevenckngaa
-datacapturegallery/ @jingtang10 @yigit @deepankarb @joiskash @kevinmost @aditya-07 @stevenckngaa
+datacapture/ @jingtang10 @yigit @deepankarb @joiskash @kevinmost @aditya-07 @stevenckngaa @ktarasenko
+datacapturegallery/ @jingtang10 @yigit @deepankarb @joiskash @kevinmost @aditya-07 @stevenckngaa @ktarasenko
```

```suggestion
catalog/ @jingtang10 @yigit @deepankarb @joiskash @kevinmost @aditya-07 @stevenckngaa @ktarasenko
```

can you update this please? we have recently changed the name.